interface/forms/procedure_order/common.php

```diff
@@ -0,0 +1,1287 @@
+                                    name='form_date_ordered'
+                                    id='form_date_ordered'
+                                    value="<?php echo $row['date_ordered']; ?>"
+                                    title="<?php xl('Date of this order', 'e'); ?>" />
+                            </div>
+                            <label for="lab_id" class="col-form-label col-md-2"><?php xl('Sending To', 'e'); ?></label>
+                            <div class="col-md-2">
+                                <select name='form_lab_id' id='form_lab_id' onchange='lab_id_changed(this)' class='form-control'>
+                                    <?php
+                                    $ppres = sqlStatement("SELECT ppid, name FROM procedure_providers ORDER BY name, ppid");
+                                    while ($pprow = sqlFetchArray($ppres)) {
+                                        echo "<option value='" . attr($pprow['ppid']) . "'";
+                                        if ($pprow['ppid'] == $row['lab_id']) {
+                                            echo " selected";
+                                            $gbl_lab = get_lab_name($pprow['ppid']);
+                                        }
+                                        echo ">" . text($pprow['name']) . "</option>";
+                                    }
+                                    ?>
+                                </select>
+                            </div>
+                            <div class="clearfix"></div>
+                        </div>
+                        <div class="form-group form-row">
+                            <label for="form_order_psc" class="col-form-label col-md-2">PSC Hold Order</label>
+                            <div class="col-md-2">
+                                <?php
+                                $pscOrderOpts = array(
+                                    'data_type' => 1,
+                                    'field_id' => 'order_psc',
+                                    'list_id' => 'boolean'
+                                );
+                                generate_form_field($pscOrderOpts, $row['order_psc']);
+                                ?>
+                            </div>
+                            <label for="form_date_collected" class="col-form-label col-md-2"><?php xl('Time Collected', 'e'); ?></label>
+                            <div class="col-md-2">
+                                <input class='datetimepicker form-control'
+                                    type='text'
+                                    name='form_date_collected'
+                                    id='form_date_collected'
+                                    value="<?php echo substr($row['date_collected'], 0, 16); ?>"
+                                    title="<?php xl('Date and time that the sample was collected', 'e'); ?>" />
```

Comment:
remove e parameter (via above)
(gonna stop pointing this one yet, rec doing this for all other places in this script)